internal/runners/export/config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+)
+
+type Config struct {
+	out output.Outputer
+}
+
+type ConfigParams struct {
+	Filters []string
+	filters []*Filter
+}
+
+type primeable interface {
+	primer.Outputer
+}
+
+func prepare(params *ConfigParams) error {
+	for _, f := range params.Filters {
+
+		filter, err := MakeFilter(f)
+		if err != nil {
+			return err
+		}
+		params.filters = append(params.filters, filter)
+	}
+
+	return nil
+}
+
+func NewConfig(prime primeable) *Config {
+	return &Config{prime.Output()}
+}
+
+func (c *Config) Run(cmd *captain.Command, params *ConfigParams) error {
+	err := prepare(params)
+	if err != nil {
+		return err
+	}
+
+	output := map[string]string{
+		Dir.String(): config.ConfigPath(),
+	}
+
+	if params.Filters == nil {
+		return c.printOutput(output)
+
+	}
```

Comment:
Rather than printing the output twice and depend on returns you can just add this conditional when you apply the filtering.